edgelet/edgelet-core/src/settings.rs

```diff
@@ -27,6 +27,34 @@ impl Connect {
     }
 }
 
+impl Default for Connect {
+    // Clippy wants us to use `option_env!("...").unwrap_or("...")` but that can't be used in consts.
+    #[allow(clippy::option_if_let_else)]
+    fn default() -> Self {
+        const DEFAULT_MANAGEMENT_URI: &str =
+            if let Some(value) = option_env!("IOTEDGE_CONNECT_MANAGEMENT_URI") {
+                value
+            } else {
+                "unix:///var/run/iotedge/mgmt.sock"
+            };
+        const DEFAULT_WORKLOAD_URI: &str =
+            if let Some(value) = option_env!("IOTEDGE_CONNECT_MANAGEMENT_URI") {
```

Comment:
This should be `IOTEDGE_CONNECT_WORKLOAD_URI`.

Reply:
Fixed.